aya-bpf-macros/src/lib.rs

```diff
@@ -73,6 +73,17 @@ pub fn xdp(attrs: TokenStream, item: TokenStream) -> TokenStream {
         .into()
 }
 
+#[proc_macro_attribute]
+pub fn xdp_frags(attrs: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
is this meant to be an alias for `#[xdp(multibuffer=true)]`? I don't think we should have two ways to do the same thing

Reply:
ooops. that's not supposed to be there - nothing to see here lol